lego/apps/companies/views.py

```diff
@@ -126,3 +141,106 @@ def get_serializer_class(self):
         elif self.action in ["create", "update", "partial_update"]:
             return CompanyInterestCreateAndUpdateSerializer
         return CompanyInterestSerializer
+
+    @action(detail=False, methods=["GET"])
+    def csv(self, *args, **kwargs):
+        user = self.request.user
+        is_admin = user.has_perm(EDIT, obj=Company)
+        if not is_admin:
+            return Response(status=status.HTTP_403_FORBIDDEN)
+
+        year = self.request.query_params.get("year")
+        semester_season = self.request.query_params.get("semester")
+        event = self.request.query_params.get("event")
+
+        try:
+            semester = Semester.objects.get(year=year, semester=semester_season)
+        except ObjectDoesNotExist:
+            return Response(status=status.HTTP_400_BAD_REQUEST)
+
+        companyInterests = CompanyInterest.objects.filter(semesters__in=[semester])
+        if event:
+            companyInterests = companyInterests.filter(events__contains=[event])
+
+        event_string = f"-{event}" if event else ""
+        response = HttpResponse(content_type="text/csv")
+        response[
+            "Content-Disposition"
+        ] = f'attachment; filename="{f"Company-interests-{year}-{semester}{event_string}"}.csv"'
+
+        writer = csv.writer(response)
+        writer.writerow(
+            [
+                "Navn på bedrift:",
+                "Kontaktperson",
+                "Mail:",
+                "Telefonnumm:",
```

Comment:
Why not Telefonnummer:

Reply:
Also, `mail` -> `e-post`. Very nitpick, but I like to be consistent.